server/controllers/auth.js

```diff
@@ -0,0 +1,26 @@
+import authService from '../services/auth';
+
+function loginUser(req, res) {
+  authService.loginUser(req.body.email, req.body)
+    .then((authObj) => {
+      return res.status(200).json(authObj);
```

Comment:
You could shorten these to 
```.then(authObj => res.json(authObj))...``` because the `res.json` will default to a 200 response